src/views/Header/Header.js

```diff
@@ -100,7 +101,7 @@ class Header extends React.PureComponent<Props, State> {
   }
 
   _navigateBack = () => {
-    this.props.navigation.goBack(null);
+    InteractionManager.runAfterInteractions(this.props.navigation.goBack);
```

Comment:
Replace this with `requestAnimationFrame` instead

Reply:
Done 👍  Though I am curious why.